extendr-api/src/robj/functions.rs

```diff
@@ -1,6 +1,86 @@
-use super::*;
+use crate::*;
+
+
+/// Create a new, empty environment parented on global_env()
+/// Use the Env{} wrapper for more detail.
+/// ```
+/// use extendr_api::*;
+/// extendr_engine::start_r();
+/// let env = new_env();
+/// assert_eq!(env.len(), 0);
+/// ```
+pub fn new_env() -> Robj {
+    call!("new.env", TRUE, global_env(), 29).unwrap()
```

Comment:
Maybe add a comment here that `29` is the default value for that parameter in `new.env()`.

Reply:
They could have chosen 42, but it is not prime!
I'll add a comment.

29 seems like a sensible default. We could make a make_sized_env() which attempts to
pick a better set of parmeters.

For example, small envs are probably better off as a unhashed.